src/main/java/com/danichef/rest/commands/AdminCommand.java

```diff
@@ -42,7 +42,7 @@ public boolean onCommand(CommandSender sender, Command command, String commandLa
 
         if ((args.length == 1) && (args[0].equalsIgnoreCase("cleanse"))) {
             player.getWorld().getEntitiesByClass(ArmorStand.class).stream()
-                    .filter(a -> a.getCustomName().equals(MessagesUtil.AS_NAME))
+                    .filter(a -> playerSit.isSeat(a))
```

Comment:
This should possibly just be a [java method reference.](https://docs.oracle.com/javase/tutorial/java/javaOO/methodreferences.html)